test/e2e/data/infrastructure-docker/bases/cluster-with-kcp.yaml

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
I would like to gitignore the generated cluster-templates and keep under source control only the bases, but it seems that this makes the apidiff batch to fail 🤔